src/content/advanced/private-clusters/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Private clusters
+description: By default, the Kubernetes API endpoint is public and the cluster has internet access. This article shows which products offer private cluster features.
+weight: 35
+menu:
+  main:
+    parent: advanced
+user_questions:
+- How do I make a workload cluster private?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+last_review_date: 2023-06-13
+---
+
+By default, the Kubernetes API endpoint is public and the cluster has internet access. You can skip this article unless you plan on creating clusters with strictly-limited networking.
+
+The following products offer private cluster features:
+
+- CAPA (AWS EC2)
+- CAPZ (Azure VMs)
```

Comment:
```suggestion
- CAPA ([Cluster API](https://docs.giantswarm.io/platform-overview/cluster-management/cluster-api/architecture/) implementation of AWS EC2)
- CAPZ ([Cluster API](https://docs.giantswarm.io/platform-overview/cluster-management/cluster-api/architecture/) implementation of Azure VMs)
```

Reply:
if we are persistent with the name then plus one. Maybe we can do this as a [glossary](https://intranet.giantswarm.io/docs/glossary/#cluster-api) term and may rely on [shortcodes](https://github.com/giantswarm/docs/tree/main/src/layouts/shortcodes) so we can easily change the term afterwards. My impression is the naming is not yet solid. A PR for unification to the Glosary asking POs for review can help :)